src/components/implementation/torrent/ramfs/ramfs.c

```diff
@@ -24,6 +25,14 @@ struct fsobj root;
 
 #define MIN_DATA_SZ 256
 
+typedef struct {
+	cbuf_t cbuf_id;
+	u32_t cbuf_len;
+	u32_t start;
+	u32_t len;
+	void* next;
+} __file_data_t;
```

Comment:
__xxx is usually used by Linux or glibc. I'm not sure what convention you should follow here, see if style guide says.